utils/systemd-redis_server.service

```diff
@@ -26,7 +26,7 @@ After=network-online.target
 [Service]
 ExecStart=/usr/local/bin/redis-server --supervised systemd --daemonize no
 ## Alternatively, have redis-server load a configuration file:
-#ExecStart=/usr/local/bin/redis-server /path/to/your/redis.conf
+#ExecStart=/usr/local/bin/server /path/to/your/server.conf
```

Comment:
I see mixed use of "server.conf" and "placeholderkv.conf" - should we consolidate on "valkey.conf"?